VideoApp/VideoApp/Video/Room/ConnectOptionsFactory.swift

```diff
@@ -35,19 +35,30 @@ import TwilioVideo
                 localVerbosity: .minimal,
                 remoteVerbosity: .minimal
             )
-            
+            builder.bandwidthProfileOptions = BandwidthProfileOptions(
+                videoOptions: VideoBandwidthProfileOptions { builder in
+                    builder.mode = .collaboration
+                    builder.maxSubscriptionBitrate = 6_000
+                    builder.maxTracks = 10
+                    builder.dominantSpeakerPriority = .high
+                    builder.trackSwitchOffMode = .detected
+                    builder.renderDimensions = VideoRenderDimensions()
+                    builder.renderDimensions?.high = VideoDimensions(width: 1_280, height: 720)
+                    builder.renderDimensions?.low = VideoDimensions(width: 352, height: 288)
```

Comment:
@aaalaniz @piyushtank @paynerc I did the best I could to make app match docs but I have the remaining open items above.

Reply:
Hey @timrozum

Thanks for getting a jump on this. Here is my feedback about the BWP default behavior. In general, I think we should follow the guidance provided for [mobile browsers](https://www.twilio.com/docs/video/tutorials/developing-high-quality-video-applications#mobile-browser-collaboration-recommended-settings) and be consistent with the mobile browser BWP settings in the react app.

1. Let's confirm with @timmydoza what the max subscription bitrate is on mobile browsers. 6000 is well under the recommended 2500000 so let's just pick the same number set in the react app. cc @ceaglest 
2. I think 10 is too large a Max Tracks on mobile. I think we should follow the mobile browser guideline and go with 5 for now.
3. Let's just not set the values for dominantSpeakerPriority, trackSwitchOffMode, and renderDimension. This will result in "Server Defaults" being used as recommended in the guide.

We should also follow the guidance and publish the camera video track with low priority. I'm not sure if that can be incorporated into this PR or needs to be tracked in a separate ticket. 

I'll let @paynerc and @piyushtank discuss the changes in the video app with respect to the encodingParameters and format dimensions.

Reply:
Discussed live.

Max subscription bitrate: Use default (2400 Kbps)

Reply:
I think I'm all set and pushed changes. They were very simple so I will plan to merge after CI passes. 

I created this ticket for the encoding parameters and format dimensions because it seems I need to discuss more and it's kind of a different area of code:

https://issues.corp.twilio.com/browse/AHOYAPPS-670